go/src/github.com/RedHatInsights/insights-ocp-controller/pkg/controller/controller.go

```diff
@@ -88,7 +88,12 @@ func (c *Controller) ScanImages() {
 
 		// Check in to schedule the scan
 		log.Printf("Checking in with Master Chief...")
-		if c.canScan(image.DockerImageMetadata.ID){
+		canScan, isHalted := c.canScan(image.DockerImageMetadata.ID)
+		if isHalted {
+			log.Printf("Chief says to HALT all further image scanning.")
```

Comment:
Why not just put this log message in canScan() and let canScan() return false, instead or returning the additional bit of state info.  Otherwise, making canScan() return some sort of enum would be better.